tests/integration/framework/util/rand.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"crypto/rand"
+)
+
+// RandomString generates a random string of length n.
+func RandomString(n int) string {
+	const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
Put it at top level, not in function.

Reply:
It is here on purpose because I didn't want to have a test specific constant be at the `util` package namespace level.

Reply:
The util is for test and we may use it in more tests.

Reply:
Until then, I’d rather we keep it inside the func to reduce noise in the util package namespace.